DnsTube/Utility.cs

```diff
@@ -14,7 +14,9 @@ public static string GetPublicIpAddress(IpSupport protocol, HttpClient Client, o
 			var maxAttempts = 3;
 			var attempts = 0;
 			errorMesssage = null;
-			var url = protocol == IpSupport.IPv4 ? "http://ipv4bot.whatismyipaddress.com" : "http://ipv6bot.whatismyipaddress.com";
+
+			var settings = new Settings();
+			var url = protocol == IpSupport.IPv4 ? settings.IPv6_API : settings.IPv6_API;
```

Comment:
You have used `settings.IPv6_API` in both places here. I assume it should be `settings.IPv4_API` for the first.